src/mainwindow.cpp

```diff
@@ -200,6 +200,22 @@ MainWindow::MainWindow(QWidget* parent)
         connect(Settings::instance(), &Settings::prettifySymbolsChanged, this, [this](bool prettifySymbols) {
             m_config->group("Settings").writeEntry("prettifySymbols", prettifySymbols);
         });
+
+        settings->setPaths(
+            m_config->group("Flamegraph").readEntry("userPaths", QDir::homePath().split(QLatin1Char(';'))),
```

Comment:
this should not be in a flamegraph group, these settings are global (and should eventually be honored in more places, outside of the flamegraph)

Reply:
it's wrong to split the home path, it's just a single path. You want to wrap it in a QStringList instead to read a list of strings